filter.c

```diff
@@ -17,6 +21,25 @@ struct bpf_luaxdp_arg {
 	__u16 length;
 } __attribute__((packed));
 
+#ifdef DOME_MODE_ROUTER
+static inline int default_action(void *data)
```

Comment:
I think this name is a bit misleading.. "default action" sounds that it might be defined as "DROP" as well.. perhaps `accept()`?